docs/documentation/schedule/examples/routes-stops-trips.md

```diff
@@ -0,0 +1,140 @@
+- The sixth field (`location_type`) is used to differentiate stops from stations
+- The first record corresponds to a bus stop, hence `location_type=0`
+- The second record corresponds to station, hence `location_type=1`
+- A complete list of values for `location_type `can be found [here](../../reference/#stopstxt).
+
+<hr>
+
+## Trips
+
+After describing the agency’s routes, it is now possible to describe the trips that are served by each route. 
+
+First, the span of service needs to be defined using [calendar.txt](../../reference/#calendartxt).
+
+[**calendar.txt**](../../reference/#calendartxt) 
+
+```
+service_id,monday,tuesday,wednesday,thursday,friday,saturday,sunday,start_date,end_date
+weekend_service,0,0,0,0,0,1,1,20220623,20220903
+```
+
+Here, a service that runs on Saturdays and Sundays only is being described, so the fields for those days are populated with 1, and the fields for the remaining days are populated with zero. This service runs from June 23 2022 until September 3 2022 as shown under the fields `start_date` and `end_date`. 
+
+In this example, the file [trips.txt](../../reference/#tripstxt) describes 3 weekend trips that are served by the MAX Orange route that was described above.
+
+[**trips.txt**](../../reference/#tripstxt) 
+
+```
+route_id,service_id,trip_id,trip_headsign,direction_id,shape_id
+303-20670,weekend_service,60270564,"MAX ORANGE SADDLETOWNE",0,3030026
+303-20670,weekend_service,60270565,"MAX ORANGE BRENTWOOD",1,3030027
+303-20670,weekend_service,60270566,"MAX ORANGE BRENTWOOD",1,3030027
+```
+
+- The `route_id` from [routes.txt](../../reference/#routestxt) that corresponds to MAX Orange is listed
+- The `service_id` from [calendar.txt](../../reference/#calendartxt) that corresponds to weekends is listed
+- Each records contains a unique ID for each trip
+The headsign text is provided, which is what is typically displayed on signs inside and outside the the bus
+- The field `direction_id` allows for a distinction between trips of the same route going in different directions. For example, distinguishing between inbound trips and outbound trips - or southbound trips and northbound trips. 
+    - In this case, trips towards Saddletowne have a `direction_id=0` and trips towards Brentwood have a `direction_id=1`. The values in direction_id have no inherent meaning, they are only used to assign one direction of travel versus another
+- The `shape_id` from [shapes.txt](../../reference/#shapestxt) that corresponds to the MAX Orange route towards Saddletowne is listed for the first record and the one for MAX Orange route towards Brentwood is listed for the second and third records
+
+
+The `shape_id=3030026` corresponds to the MAX Orange towards Saddletowne. The file below includes information on the points that outline the route as well as the distance between those points. With this information, it is possible to plot the route on a map for trip planning or analytics purposes.
```

Comment:
> The file below includes information on the points that outline the route as well as the distance between those points.

- points that outline the "trip"
- "distance between those point" may confuse ppl? (`shape_dist_traveled` is for the distance from the first shape point to the point specified in this record)

Reply:
Updated the text to: "The file below includes information on the points that outline the trip as well as the distance between each point and the start of the trip."